api/lib/infrastructure/repositories/complementary-certification-repository.js

```diff
@@ -19,4 +19,12 @@ const getByLabel = async function ({ label }) {
   return _toDomain(result);
 };
 
-export { findAll, getByLabel };
+const getById = async function ({ complementaryCertificationId }) {
```

Comment:
On est dans le repo des complementary certifications, on pourrait simplifier:
```suggestion
const getById = async function (id) {
```